REST-API-ADVANCED/advanced/src/main/java/com/epam/esm/hateoas/HateoasSupport.java

```diff
@@ -0,0 +1,165 @@
+package com.epam.esm.hateoas;
+
+import com.epam.esm.certificate.controller.CertificateController;
+import com.epam.esm.certificate.entity.Certificate;
+import com.epam.esm.order.controller.OrderController;
+import com.epam.esm.order.entity.Order;
+import com.epam.esm.tag.controller.TagController;
+import com.epam.esm.tag.entity.Tag;
+import com.epam.esm.user.controller.UserController;
+import com.epam.esm.user.entity.User;
+import org.springdoc.api.annotations.ParameterObject;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.domain.Pageable;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder;
+
+import java.util.List;
+
+@Configuration
+public class HateoasSupport {
```

Comment:
I would like you to split this class into separate ones for each entity

Reply:
I'll try to rework it tomorrow

Reply:
done